### ISSUE TITLE
New note function doesn't work

### ISSUE THREAD

**Issue by arBorghi** [shabegom/buttons#54]
I dunno why, but when i try creating a new note from a template it doesn't work at all.
That's the code generated by Button Maker:

```button
name Nueva idea
type note(Nueva idea) template
action 📜 NUEVA Idea
color purple
```
^button-ocsm

**Comment by mnvwvnm** [comment]
Same problem here. It worked, two weeks ago. But not now!

**Comment by shabegom** [comment]
I'll release a bug fix for this on the weekend. Sorry!

**Comment by shabegom** [comment]
Got to it this morning! Should be fixed in 0.4.11
